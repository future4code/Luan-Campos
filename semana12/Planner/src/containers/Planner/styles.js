import styled from "styled-components";

export const Wrapper = styled.div`
  text-align: center;
`;

export const Form = styled.form``;

export const WrapperWeek = styled.div`
  width: 1400px;
  display: flex;
  justify-content: center;
  align-items: center;
  border: 1px solid black;
  margin: 0 auto;

  ul {
    width: 100%;
    display: flex;
    flex-direction: column;

    li {
      list-style: none;
      width: 100px;
      border: 1px solid blue;
      margin-top: 10px;
    }
  }
`;

export const dias = [
  "Segunda",
  "Terça",
  "Quarta",
  "Quinta",
  "Sexta",
  "Sábado",
  "Domingo",
];