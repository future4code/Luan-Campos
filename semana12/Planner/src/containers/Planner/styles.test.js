import React from "react";
import renderer from "react-test-renderer";
import { Wrapper, Form, WrapperWeek, dias } from "./styles";

it("Estilização", () => {
  const tree = renderer
    .create(<Wrapper />, <Form />, <WrapperWeek />, dias)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
