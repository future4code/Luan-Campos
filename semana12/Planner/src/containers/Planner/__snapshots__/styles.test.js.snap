// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Estilização 1`] = `
<div
  className="sc-bdVaJa khUSez"
/>
`;
